src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarEntry.cs

```diff
@@ -286,6 +286,7 @@ public Task ExtractToFileAsync(string destinationFileName, bool overwrite, Cance
         /// </summary>
         /// <remarks>
         /// If the entry does not come from an archive stream or if the archive stream is not seekable, returns -1.
+        /// The position value returned by this property is relative to the absolute start of the archive stream, independently of where the tar archive beging.
```

Comment:
```suggestion
        /// The position value returned by this property is relative to the absolute start of the archive stream, independent of where the tar archive begins.
```

Reply:
This is expected because my English starts to atrophy on Fridays.